packages/slot/src/slot-manager.js

```diff
@@ -121,6 +121,16 @@ class SlotManager {
     return Object.keys(this.intents[intent]);
   }
 
+  /**
+   * Given an intent return the information if the intent had entities defined
+   *
+   * @param {String} intent Name of the intent.
+   * @returns {boolean} true if intent has defined entities, else false
+   */
+  intentHasEntities(intent) {
```

Comment:
maybe better some method name starting by verb like hasIntentEntities to keep consistence with other method names

Reply:
done